attributes/composer.rb

```diff
@@ -0,0 +1 @@
+default['composer']['url'] = 'https://getcomposer.org/download/1.2.1/composer.phar'
```

Comment:
Please move this to the composer recipe and use ``node[foo]`` instead of default.